app/models/notification_services/hipchat_service.rb

```diff
@@ -38,18 +42,34 @@ def url
     end
 
     def create_notification(problem)
-      url = app_problem_url problem.app, problem
-      message = <<-MSG.strip_heredoc
-        <strong>#{ERB::Util.html_escape problem.app.name}</strong> error in <strong>#{ERB::Util.html_escape problem.environment}</strong> at <strong>#{ERB::Util.html_escape problem.where}</strong> (<a href="#{url}">details</a>)<br>
-        &nbsp;&nbsp;#{ERB::Util.html_escape problem.message.to_s.truncate(100)}<br>
-        &nbsp;&nbsp;Times occurred: #{problem.notices_count}
-      MSG
+      format = self[:mentions].present? ? "text" : "html"
```

Comment:
I'm pretty sure everyone will forget why this is necessary. Could you add your note in a comment here?